engine/src/main/java/com/odysseusinc/arachne/executionengine/execution/r/RService.java

```diff
@@ -76,26 +73,16 @@ public Overseer analyze(AnalysisSyncRequestDTO analysis, File analysisDir, BiCon
         File keystoreDir = new File(analysisDir, "keys");
         KrbConfig krbConfig = kerberosSupport.getConfig(analysis, keystoreDir);
 
-        DescriptorBundle bundle = descriptorService.getDescriptorBundle(analysisDir,
```

Comment:
Where did you move this logging logic below?

It seems to be to TarballRService

Reply:
This logic is specific to tarball version mode. This is superclass is used for both tarball and docker, so it should not be here. Placing it in this class was an abstraction failure mistake.